tests/scalers/redis/redis_cluster_streams_pending/redis_cluster_streams_pending_test.go

```diff
@@ -0,0 +1,234 @@
+//go:build e2e
```

Comment:
Rename path to: tests/scalers/redis/redis_cluster_streams/redis_cluster_streams_pending_entries_test.go

Notice that the folder name (redis_cluster_streams) is not changing and only the file name is.